Scripts/Miscellaneous/Spotify_Playlist_Generator/spotify_playlist_generator.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+import os
+import random
+
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+scope = "playlist-modify-public"
+
+# Create an instance of spotify library
+spotipy_instance = spotipy.Spotify(auth_manager=SpotifyOAuth(scope=scope, cache_path=os.getcwd()))
+
+
+# Method to read the command line arguments and parse them
```

Comment:
instead of adding comments outside from functions make doc string inside in functions
```suggestion
def get_args():
     """Method to read the command line arguments and parse them
     
     Input:
     Output:
     """
```